### ISSUE TITLE
block place postition 

### ISSUE THREAD

**Issue by lllWinchesterlll** [maruohon/litematica#328]
is there any way to click on the block and put it in the correct schematic position?

**Comment by maruohon** [comment]
It sounds like you want to use the Easy Place mode? Using that you can just right click on the schematic blocks to place them. It does click on the target position's air block, so some spigot/paper servers might not allow that.

**Comment by lllWinchesterlll** [comment]
yes, I use this function, but I wanted the block to be in the correct orientation even if I turned the wrong way!

**Comment by maruohon** [comment]
It does try to auto-rotate the blocks, but it currently needs another mod to handle that request. In single player simply having Tweakeroo installed does that, on a server you would currently need CarpetExtra (and FabricCarpet) and the `accurateBlockPlacement` rule enabled. And also not all blocks are supported yet in the protocol.
